package.json

```diff
@@ -9,7 +9,7 @@
     "npm": "8.5.5"
   },
   "devDependencies": {
-    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git",
+    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#task/FP-1378-consistent-space-above-footer",
```

Comment:
This change is not intended to be merged. After PR approval, I plan to revert this change so the latest Core-Styles is loaded.

<details>

That branch is transient, so merging this to `main` would soon result in `npm install` fail. What I should do, is pin to a specific version… but I am currently loading latest of core-styles, to facilitate rapid development. It **is** versioned ([releases](https://github.com/TACC/Core-Styles/releases/)). Also, I don't really know how to make a process for Core-Styles, yet. How do I version what hasn't been merged?

</details>